src/components/content-region/index.jsx

```diff
@@ -20,11 +20,19 @@ const COMPONENTS = {
 	}
 };
 
-for (let i = 1; i <= 6; i++) {
-	const Tag = 'h' + i;
+// Skip h1, no need to link to a page title
+for (let i = 2; i <= 6; i++) {
+	const Tag = /** @type {keyof preact.JSX.IntrinsicElements} */ ('h' + i);
 	COMPONENTS[Tag] = function header(props) {
 		props.children = props.id
-			? [<a href={'#' + props.id} />, props.children]
+			? [
+				<a href={'#' + props.id}>
+					<svg width="16" height="16" viewBox="0 0 24 24" aria-label={`Link to: ${props.children} (#${props.id})`}>
```

Comment:
This implementation (e.g., `Link to: Setting up JSX (#setting-up-jsx)`) came from Vue's docs which I quite like, makes it quite clear why there's a link right alongside the header content.